eCourse_backend/file_exchange/models.py

```diff
@@ -1,16 +1,35 @@
+import os
+from django.core.exceptions import ValidationError
+from django.utils import timezone
 from django.db import models
 from users.models import User
 from courses.models import Exercise, Course
 
+# upload dir
+
 
 def exercise_directory_path(instance, filename):
-    return '/upload/course_{0}/exercise_{1}/{2}'.format(
+    return './upload/course_{0}/exercise_{1}/{2}'.format(
```

Comment:
Instead of using a dot, removing the first slash _should_ have the same effect

Reply:
yes you're right